jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -30,15 +41,87 @@
             persistent-hint>
           </v-switch>
         </v-row>
+        <div v-if="link_type_selected == 'WCS'">
+
+          <j-plugin-section-header>Select orientation</j-plugin-section-header>
+          <plugin-viewer-select
+            :items="viewer_items"
+            :selected.sync="viewer_selected"
+            :multiselect="multiselect"
+            :label="multiselect ? 'Viewers' : 'Viewer'"
+            :show_if_single_entry="multiselect"
+            :hint="multiselect ? 'Select viewers to set options simultaneously' : 'Select the viewer to set orientation'"
+          />
+          <plugin-layer-select
+            :items="layer_items"
+            :selected.sync="layer_selected"
+            :multiselect=false
+            :show_if_single_entry="true"
+            :label="'Orientation in viewer'"
+            :hint="'Select the viewer orientation'"
+          />
+          <v-row>
+            <span style="line-height: 36px">Presets:</span>
+            <j-tooltip tooltipcontent="Default orientation">
+              <v-btn icon @click="select_default_orientation">
+                <v-icon>mdi-restore</v-icon>
+              </v-btn>
+            </j-tooltip>
+            <j-tooltip tooltipcontent="north up, east left">
+              <v-btn icon @click="set_north_up_east_left">
+                <img :src="icon_nuel" width="24" class="invert-if-dark" style="opacity: 0.65"/>
+              </v-btn>
+            </j-tooltip>
+            <j-tooltip tooltipcontent="north up, east right">
+              <v-btn icon @click="set_north_up_east_right">
+                <img :src="icon_nuer" width="24" class="invert-if-dark" style="opacity: 0.65"/>
+              </v-btn>
+            </j-tooltip>
+          </v-row>
+        </div>
+
+        <div style="grid-area: 1/1">
+        </div>
+        <div v-if="link_type_selected == 'WCS'">
+
+          <j-plugin-section-header>Add orientation options</j-plugin-section-header>
+
+              <plugin-auto-label
+                :value.sync="new_layer_label"
+                :default="new_layer_label_default"
+                :auto.sync="new_layer_label_auto"
+                label="Name for orientation option"
+                hint="Label for this new model component."
+              ></plugin-auto-label>

```

Comment:
elsewhere we put this at the end all the input options, so we might want to consider that here as well.

Reply:
Good call! Sounds/looks good:

<img width="341" alt="Screen Shot 2023-08-02 at 12 17 11" src="https://github.com/bmorris3/jdaviz/assets/3497584/f0824cc7-18fb-4f26-8cf0-771be072930e">